plugins/home/src/plugin.ts

```diff
@@ -210,3 +211,18 @@ export const HomePageRecentlyVisited = homePlugin.provide(
       import('./homePageComponents/VisitedByType/RecentlyVisited'),
   }),
 );
+
+/**
+ * A component to display specific Featured Docs.
+ *
+ * @public
+ */
+export const FeaturedDocs = homePlugin.provide(
+  createReactExtension({
+    name: 'FeaturedDocs',
+    component: {
+      lazy: () =>
+        import('./homePageComponents/FeaturedDocs').then(m => m.FeaturedDocs),
```

Comment:
Do you think it might make sense to name this one FeaturedDocsCard (and FeaturedDocsCardProps)? most cards have that suffix. Same for the actual exported variable name on line 220 above of course